### ISSUE TITLE
Skip kc-info-message and proceed directly to password update form

### ISSUE THREAD

**Issue by hookenful** [InseeFrLab/keycloakify#94]
Hello there!
 
I have a question regarding that kc-info-message component which locates in `Info.tsx` in your components.
I am sending an email to user with executing action link. After he clicks it redirect to modal:
 Perform the following action(s): **Update password**

 Is it possible to skip it and proceed directly to password update form?

![image](https://user-images.githubusercontent.com/31849464/165336948-ddf7e4e1-aa89-4720-8aaf-04e29aff025b.png)

Yes, I know it is not initialized through keycloakify yet, but I just wanted to ask a question before proceeding to code.

Thanks in advance.

**Comment by garronej** [comment]
Hi @hookenful,  
Sure thing,  
Keycloakify give you complete control as long as you respect the flow.  
You could have a useEffect that would programatically trigger a click on the link or even better doing `window.location = kcContext.url.<the url to procede>`.  
It's up to you really.

**Comment by hookenful** [comment]
Thank you, I'll try that, but before it, I faced to a problem with white screen with new version of theme created by our frontend. Can you check the symptoms please?

Screen 1: working theme before change

![image](https://user-images.githubusercontent.com/31849464/165711901-8f07efac-5243-4781-8c3f-2c5439d722bb.png)

Screen 2: after I just changed mounted theme to the new one.

![image](https://user-images.githubusercontent.com/31849464/165712532-9c079d0d-19d8-4f01-a3ae-d32d4b22d3f5.png)

**Comment by garronej** [comment]
You mean a problem appeared with the new keycloak version I released in the past 24h?  
Which was the last version working as expected?

**Comment by hookenful** [comment]
No, it is mine fault. Or keycloak's, because I store /keycloak.json in public folder with web app, but keycloak want it to find in his /public..

**Comment by garronej** [comment]
Ok.
You'll have to show me some more code.
How are you accessing keycloak.json?

**Comment by hookenful** [comment]
I got it working, but I think now there is a problem with kcContext.urls, there are not "urlS" but url.
Look at code:

![image](https://user-images.githubusercontent.com/31849464/165738657-c652c9b7-fa74-4719-a927-71c9b0bb12ba.png)

![image](https://user-images.githubusercontent.com/31849464/165750039-7d0a4b3a-5bdb-48e6-9a05-0902e42d9de8.png)

**Comment by hookenful** [comment]
Can't point keycloakify to determine right url for action

**Comment by garronej** [comment]
>  there are not "urlS" but url.  

Absolutely, take the snippet of code I shared previously as psudo code. It was what came at the top of my head but you should trust the code definitions.  
I see you are customizing at the component level. If all you want to acheive is skipping kc-info-message you can go CSS only.  

The url you are looking for is `kcContext.actionUri`  

<img width="1530" alt="image" src="https://user-images.githubusercontent.com/6702424/165753699-08e95cc5-5206-4a33-8a6f-117b9d01f77e.png">

<img width="703" alt="image" src="https://user-images.githubusercontent.com/6702424/165753739-20179fa6-5c50-4e8f-ae2c-7d546139422d.png">

Put this literally anywhere in your code, just after `getKcContext` for example
```ts
if( kcContext?.pageId === "info.ftl" && kcContext.actionUri !== undefined ){
    document.location.replace(kcContext.actionUri)
}
```

**Comment by hookenful** [comment]
Finally, we made it to work. Now all is ok, if something changes, I'll be in touch. Thanks again and good luck :)

~ We've got confused because being ping-ponged frontend keycloak integration problems through our frontend developer, but I am backend maintaner)
